website/docs/data-display/bubble-chart/examples/legend.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  Plot,
+  Bubble,
+  XAxis,
+  YAxis,
+  LegendItem,
+  ChartLegend,
+  makeDataHintsContainer,
+} from '@semcore/ui/d3-chart';
+import { scaleLinear } from 'd3-scale';
+import { Text } from '@semcore/ui/typography';
+import { Flex } from '@semcore/flex-box';
+
+const dataHints = makeDataHintsContainer();
+
+export default () => {
+  const MARGIN = 40;
+  const width = 500;
+  const height = 300;
+
+  const xScale = scaleLinear()
+    .range([MARGIN, width - MARGIN])
+    .domain([0, 10]);
+
+  const yScale = scaleLinear()
+    .range([height - MARGIN, MARGIN])
+    .domain([0, 10]);
+
+  const [legendItems, setLegendItems] = React.useState(
```

Comment:
`setLegendItems` is unused. Do you really need state in this example?

Reply:
copy-paste. removed